README.md

```diff
@@ -134,7 +134,7 @@ var colorCursor = paletteCursor.select('colors');
 
 #### Updates
 
-A *baobab* tree can obviously be updated. However, one has to understand that he won't do it, at least by default, synchronously.
+A *baobab* tree can obviously be updated. However, one has to understand that they won't do it, at least by default, synchronously.
```

Comment:
Shouldn't we use `it`? I am not sure whether this is the same for English as for my native language but I am referring to the Baobab tree in this sentence, not the user.